.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: Select Xcode
-      run: sudo xcode-select -switch /Applications/Xcode_11.2.app
+      run: sudo xcode-select -switch /Applications/Xcode_11.2.1.app && /usr/bin/xcodebuild -version
```

Comment:
Also change the destination to reflect the new OS version.

Reply:
Good point. :)